server/game/cards/04.1-FJ/Confession.js

```diff
@@ -0,0 +1,65 @@
+const GameActions = require('../../GameActions/index.js');
+const SpellCard = require('../../spellcard.js');
+
+class Confession extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Noon: Miracle 6',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            difficulty: 6,
+            target: {
+                activePromptTitle: 'Select a dude to add bounty to',
+                waitingPromptTitle: 'Waiting for opponent to select a dude',
+                cardCondition: { 
+                    location: 'play area', 
+                    condition: card =>
+                        card.gamelocation === this.gamelocation || 
+                        card.getGameLocation().isAdjacent(this.gamelocation)
+                },
+                cardType: ['dude'],
+                gameAction: 'addBounty'
+            },
+            onSuccess: (context) => {
+                this.game.resolveGameAction(GameActions.bootCard({ card: this.parent }), context);
+                this.game.resolveGameAction(GameActions.addBounty({
+                    card: context.target, amount: this.parent.getSkillRating('blessed')
+                }));
+                this.game.addMessage('{0} uses {1} to boot {2} and add bounty to {3}', context.player, this, this.parent, context.target);
+            },
+            source: this
+        });
+
+        this.spellAction({
+            title: 'Noon: Miracle 7',
```

Comment:
```suggestion
            title: 'Noon: Transfer bounty from a dude',
```
I would use more descriptive action titles.